src/metric-engine/src/engine/create.rs

```diff
@@ -381,6 +381,10 @@ impl MetricEngineInner {
         // concat region dir
         let metadata_region_dir = join_dir(&request.region_dir, METADATA_REGION_SUBDIR);
 
+        // remove TTL option
+        let mut options = request.options.clone();
+        options.remove("ttl");
```

Comment:
Perhaps it's better to throw an error message than silently delete?

Reply:
This is not an error case. TTL still applies for data regions.